```
~ awk --version
awk version 20121220

~ make generate
[...]
# Add comment recognized by reviewable.
echo '// GENERATED FILE DO NOT EDIT' > cockroach/pkg/ui/src/js/protos.d.ts
pbts ./cockroach/pkg/ui/src/js/protos.js >> cockroach/pkg/ui/src/js/protos.d.ts
set -eo pipefail; ret=`goyacc -o irgen.go.tmp -p irgen irgen.y`; \
          ! echo "$ret" | grep -F conflicts
# First comment for github/Go; second for reviewable.
# https://github.com/golang/go/issues/13560#issuecomment-277804473
# https://github.com/Reviewable/Reviewable/wiki/FAQ#how-do-i-tell-reviewable-that-a-file-is-generated-and-should-not-be-reviewed
(echo '// Code generated by goyacc. DO NOT EDIT.'; \
         echo '// GENERATED FILE DO NOT EDIT'; \
         cat irgen.go.tmp) >irgen.go && rm -f irgen.go.tmp
mkdir -p gen
awk -f all_keywords.awk < sql.y > keywords.go.tmp || rm keywords.go.tmp
awk -f reserved_keywords.awk < sql.y > reserved_keywords.go.tmp || rm reserved_keywords.go.tmp
awk -f help.awk < sql.y > help_messages.go.tmp || rm help_messages.go.tmp
set -euo pipefail; \
        TYPES=$(awk '/func.*sqlSymUnion/ {print $(NF - 1)}' sql.y | sed -e 's/[]\/$*.^|[]/\\&/g' | tr '\n' '|' | sed -E '$s/.$//'); \
        sed -E "s_(type|token) <($TYPES)>_\1 <union> /* <\2> */_" < sql.y | \
        awk -f replace_help_rules.awk > gen/sql.y
mv -f help_messages.go.tmp help_messages.go
mv -f keywords.go.tmp keywords.go
mv -f reserved_keywords.go.tmp reserved_keywords.go
awk: weird printf conversion %p
 input record number 884, file
 source line number 19
awk: not enough args in printf(%s %prec VALUES | %s HELPTOKEN %%prec UMINUS { return helpWith(sqllex, "%s") }
)
 input record number 884, file
 source line number 19
make[1]: *** [gen/sql.y] Error 2
make[1]: *** Waiting for unfinished jobs....
[...]
```

--- a/pkg/sql/parser/replace_help_rules.awk
+++ b/pkg/sql/parser/replace_help_rules.awk
@@ -16,7 +16,7 @@
         #    ^^^^^^^^^^^^^^^^^^^ take this
         rulename = substr(rulename, index($0, "|")+1)
     }
-    printf "%s %prec VALUES | %s HELPTOKEN %%prec UMINUS { return helpWith(sqllex, \"%s\") }\n", prefix, rulename, helpkey
+    printf "%s %%prec VALUES | %s HELPTOKEN %%prec UMINUS { return helpWith(sqllex, \"%s\") }\n", prefix, rulename, helpkey
     next
 }
 /\/\/ SHOW HELP:/ {

